src/chum.cpp

```diff
@@ -60,6 +62,26 @@ void Chum::setShowAppsByDefault(bool v) {
   emit showAppsByDefaultChanged();
 }
 
+void Chum::setManualVersion(const QString &v) {
+    if (!m_ssu.manageRepo()) {
+        emit error(qtTrId("chum-repo-management-disabled-title"));
+        return;
+    }
+
+    if (m_manualVersion == v) return;
+    m_manualVersion = v;
```

Comment:
```suggestion
    m_manualVersion = v.trimmed();
```

This is to avoid accidental spaces in text field